package.json

```diff
@@ -35,6 +35,9 @@
     "type": "git",
     "url": "https://github.com/web-infra-dev/rspack"
   },
+  "engines": {
+    "pnpm": "7.32.0"
```

Comment:
Please update to the latest version of pnpm.

Reply:
that's another thing, we should upgrade to 8.x when we merge 0.4.0 back and remove NodeJS 14 support